packages/truffle/cli.webpack.config.js

```diff
@@ -8,37 +8,25 @@ const outputDir = path.join(__dirname, "build");
 
 module.exports = {
   entry: {
-    cli: path.join(
-      __dirname,
-      "../..",
-      "node_modules",
-      "truffle-core",
-      "cli.js"
-    ),
+    cli: path.join(__dirname, "../..", "node_modules", "core", "cli.js"),
```

Comment:
Believe all of these need to be like...

```suggestion
    cli: path.join(__dirname, "../..", "node_modules", "@truffle/core", "cli.js"),
```

Reply:
Yeah that totally makes sense, thanks!